### PR TITLE
docs(ja): translate guides/configuration-glossary/configuration-modern

### PR DESCRIPTION
**Pull request by aytdm**
@inouetakuya @53able please review this PR.

#### Ref

- https://ja.nuxtjs.org/api/configuration-modern
- https://github.com/nuxt/nuxtjs.org/issues/544
- https://github.com/vuejs-jp/ja.nuxtjs.org/issues/22

### CODE REVIEW

File: content/ja/guides/configuration-glossary/configuration-modern.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+category: configuration-glossary
+position: 18
+---
+
+> この機能は [vue-cli の modern mode](https://cli.vuejs.org/guide/browser-compatibility.html#modern-mode) にインスパイアされています
+
+- 型: `String` または `Boolean`
+  - デフォルト: false
+  - 設定できる値:
+    - `'client'`: モダンバンドル `<script type="module">` とレガシーバンドル`<script nomodule>` スクリプトの両方を提供し、モダンバンドル用の `<link rel="modulepreload">` も提供します。`module` タイプを理解するすべてのブラウザではモダンバンドルをロードしますが、古いブラウザではレガシー（トランスパイルされた）バンドルにフォールバックします。
```
Comment (inouetakuya) [inline content/ja/guides/configuration-glossary/configuration-modern.md:14]: 👁️ 

```suggestion
    - `'client'`: モダンバンドル `<script type="module">` とレガシーバンドル `<script nomodule>` スクリプトの両方を提供し、モダンバンドル用の `<link rel="modulepreload">` も提供します。`module` タイプを理解するすべてのブラウザではモダンバンドルをロードしますが、古いブラウザではレガシー（トランスパイルされた）バンドルにフォールバックします。
```

**Review by inouetakuya — APPROVED** [review]
あ、間違えて Approve してしまいました！ :pray:  

1箇所、細かいですが修正お願いします！

**Review by inouetakuya — CHANGES_REQUESTED** [review]
誤って Approve してしまったため 🙏

**Comment by aytdm** [comment]
ありがとうございます。修正しました

**Review by inouetakuya — APPROVED** [review]
Thank you!! 👍

**Comment by inouetakuya** [comment]
@pi0 @debs-obrien 

https://github.com/nuxt/nuxtjs.org/pull/755#issuecomment-712346353

I have gotten a invitation mail of this repository from @pi0, however I don't see have write access to merge this PR.

My view:

image

**Comment by debs-obrien** [comment]
@inouetakuya can you check now if you have permissions to merge. Thanks

**Comment by inouetakuya** [comment]
@debs-obrien CC: @pi0 @aytdm 

It looks that I can merge this PR!! Thank you very much!! ☺️ 

image

**Pull request merged**